### PR TITLE
sig-release/charter: Update link to KEP directory

### PR DESCRIPTION
**Pull request by PurneswarPrasad**
**Which issue(s) this PR fixes**:

A broken link to KEP template in the charter.md documentation is fixed here.

### CODE REVIEW

**Comment by reylejano** [comment]
Thank you for updating the link
/lgtm

File: sig-release/charter.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Additional requirements:
 - [sigs.yaml] must be updated to include subproject information and OWNERS files with subproject chairs
 
 
-[KEP]: https://git.k8s.io/enhancements/keps/YYYYMMDD-kep-template.md
+[KEP]: https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md
```
Comment (sftim) [inline sig-release/charter.md:99]: How about linking to https://git.k8s.io/enhancements/keps ?

Comment (PurneswarPrasad) [reply sig-release/charter.md:99]: The previous link was done to the template of a KEP. Surely, linking it to the whole repo would be very useful, but I'm not sure if we really need that.

Comment (sftim) [reply sig-release/charter.md:99]: https://git.k8s.io/enhancements/keps is in fact the index of KEPs and also a page that explains what a KEP is; that's why I suggested linking there.

Comment (PurneswarPrasad) [reply sig-release/charter.md:99]: Ahh yes, that makes sense. I'll do it, thanks!

**Review by justaugustus — APPROVED** [review]
/lgtm
/approve
/retitle sig-release/charter: Updated link to KEP directory

**Comment by justaugustus** [comment]
/retitle sig-release/charter: Update link to KEP directory

**Pull request merged**